### PR TITLE
Fixed incorrect rendering of Logical OR and Bitwise OR operator

### LINKED ISSUE #5527: Incorrect rendering of pipes in Operators table
### Page URL

https://dart.dev/language/operators

### Page source

_No response_

### Describe the problem

Bitwise OR & logical OR operator isn't being rendered properly
Screenshot 2024-02-10 203124

### Expected fix

_No response_

### Additional context

_No response_

### I would like to fix this problem.

- [X] I will try and fix this problem on dart.dev.

### PR DESCRIPTION
**Pull request by iambijayd**
Fixes #5527
---

- [x] I’ve reviewed the contributor guide and applied the relevant portions to this PR.
- [x] This PR doesn't contain automatically generated corrections or text (Grammarly, LLMs, and similar).
- [x] This PR follows the [Google Developer Documentation Style Guidelines](https://developers.google.com/style) — for example, it doesn't use _i.e._ or _e.g._, and it avoids _I_ and _we_ (first person).
- [x] This PR uses [semantic line breaks](https://github.com/dart-lang/site-shared/blob/main/doc/writing-for-dart-and-flutter-websites.md#semantic-line-breaks) of 80 characters or fewer.

### CODE REVIEW

File: src/content/language/operators.md [hunk 1]
```diff
@@ -14,29 +14,29 @@ nextpage:
...
+| shift                                   | `<<`    `>>`    `>>>`                                                                                                                                                                              | Left          |
+| bitwise AND                             | `&`                                                                                                                                                                                                | Left          |
+| bitwise XOR                             | `^`                                                                                                                                                                                                | Left          |
+| bitwise OR                              | `                                                                                                                                                                                                  | `             | Left |
```
Comment (parlough) [inline src/content/language/operators.md:31]: It looks like this entry will need to be fixed as well to: `<code>&#124;</code>

**Review by parlough — CHANGES_REQUESTED** [review]
Thanks for reporting and fixing this!

Your PR included a lot of formatting changes to the page, making it harder to review, but also breaking some of our tooling. Can you update the PR to instead just change the lines that need to be updated?

File: src/content/language/operators.md (hunk 1 above)
Comment (iambijayd) [reply src/content/language/operators.md:31]: Had some issues with previous branch. I have opened a new pull request. Can you review that?

**Pull request closed without merging**